Source/Adobe.Target.Client/Model/TargetDeliveryRequest.cs

```diff
@@ -524,12 +539,26 @@ private void CreateAndSetAnalyticsValues()
                     TrackingServer = this.TrackingServer,
                     TrackingServerSecure = this.TrackingServerSecure,
                     Logging = LoggingType.ServerSide,
-                    SupplementalDataId = this.Visitor.GetSupplementalDataId(TargetConstants.DefaultSdidConsumerId),
+                    SupplementalDataId = this.Visitor.GetSupplementalDataId(this.GetVisitorConsumerId()),
                 };
 
                 this.ExperienceCloud.Analytics = analyticsRequest;
             }
 
```

Comment:
Can't we use a simpler hash function here? If we want to have the same logic across SDKs, let's keep this as simple as possible.

Reply:
I have used the simplest approach here, adding zero code, as we already have the Murmurhash32 implementation across all our SDKs. Murmurhash is a great fit for this use-case, see https://stackoverflow.com/a/42073872
We do need a **non-crypto** hash for this use-case, and we shouldn't use any of the crypto hashes like MD5, SHA-1, SHA256 as these may be flagged during security audits.
We could use an alternative hashing algorithm, like [FNV-1a](http://isthe.com/chongo/tech/comp/fnv/) for example, but these would add code to the web SDKs, so I think just reusing Murmurhash is the best option.

Reply:
Fixed by porting a simple hash implementation from the web SDK